src/gazebo2rviz/gazebo2moveit_collision_node.py

```diff
@@ -10,6 +10,8 @@
 import rospy
 from tf.transformations import *
 from sdf2moveit_collision_node import Sdf2moveit
+from smart_manipulation_framework_core.srv import GetGazeboType
```

Comment:
this is a generic package and would be best if it's independent of any specific shadow repos if possible. At least, it should be one of the common packages that is public.